gnucash/report/reports/standard/advanced-portfolio.scm

```diff
@@ -129,7 +129,7 @@ by preventing negative stock balances.<br/>")
       (filter gnc:account-is-stock?
               (gnc-account-get-descendants-sorted
                (gnc-get-current-root-account)))
-      (list ACCT-TYPE-ASSET ACCT-TYPE-BANK ACCT-TYPE-STOCK ACCT-TYPE-MUTUAL))
+      (list ACCT-TYPE-ASSET ACCT-TYPE-STOCK ACCT-TYPE-MUTUAL))
```

Comment:
You are accidentally reverting it the last commit to advanced-portfolio.scm here accidentally. Most likely due to your struggle with rebasing to stable. https://github.com/Gnucash/gnucash/commit/aca602103e113c6e0d33ad30e8d11a0926ed5eba